test/tools/webpack-test-server.ts

```diff
@@ -1,38 +1,32 @@
 import {IMochaServer} from 'mocha-webdriver';
-import * as merge from 'webpack-merge';
-import * as serve from 'webpack-serve';
+import * as webpack from 'webpack';
+import * as WebpackDevServer from 'webpack-dev-server';
 
 // tslint:disable:no-console
 
 export class WebpackServer implements IMochaServer {
-  // The result of webpack-serve call. See https://github.com/webpack-contrib/webpack-serve#serveargv-options
+  // WebpackDevServer instance. See https://github.com/webpack/docs/wiki/webpack-dev-server
   private _server: any;
+  private _port: number = 0;
 
   public async start() {
     const config = require('../fixtures/webpack.config.js');
-    console.log("Starting webpack-serve");
-    this._server = await serve({}, {
-      config: merge(config, {
-        serve: {
-          port: 9010,
-          open: false,
-          clipboard: false,
-          hotClient: false,
-          logLevel: 'warn',
-        },
```

Comment:
Are these options no longer needed?

Reply:
they are specific to webpack-serve. For webpack-dev-server, it doesn't seem so many options are needed.